### ISSUE TITLE
[bug] Immersive Engineering compatibility not working.

### ISSUE THREAD

**Issue by DigitalData** [Darkhax-Minecraft/BotanyPots#353]
- Minecraft 1.20.1
- Forge 47.2.17
- Immersive Engineering 10.0.0-169
- Botany Pots 13.0.17
This is part of a custom modpack.

Hello, 

looks like the seeds from immersive engineering are not recognised by botany pots as of the above version. Recipes don't show up in JEI either.
Is this a bug, or do I need to create a data pack using the commit referenced in earlier issues?

Cheers for the help.

**Comment by DigitalData** [comment]
Could this be related to the fact that IE switched to NeoForged-only compatibility? I'm still loading it through Forge, but I'm wondering if that could be the cause.

**Comment by DigitalData** [comment]
~~The [bookshelf conditional load](https://github.com/Darkhax-Minecraft/BotanyPots/blob/d7fb4a1ae530c1fa2f84afa41f3c6c34fd375f11/common/src/main/resources/data/botanypots/recipes/immersiveengineering/crop/hemp.json#L2) was not detecting the `immersiveengineering:seed` item. My theory is that this has been causing trouble since IE switched from Forge to NeoForge.~~

~~I'll create an issue on the bookshelf repo as well.~~

~~After creating my own data pack, which copies what you have but without the bookshelf load condition, and recipes for IE and botany pots are now showing up.~~

_Edit: this was incorrect, please ignore._

**Comment by DigitalData** [comment]
Update on this issue. Turns out the above was incorrect. Looks like replacing the `display` entry with 
```
  "display": {
    "block": "immersiveengineering:hemp"
  },
```
Fixed the recipe. Now I reckon the block animations aren't being detected correctly. I'm not invested enough in the aesthetic to investigate further.

**Comment by DigitalData** [comment]
Using
```
  "display": {
    "type": "botanypots:aging",
    "block": "immersiveengineering:hemp"
  },
```
Seems to be working

**Comment by Darkhax** [comment]
This issue was related to Immersive Engineering standardizing their blockstate property names causing BotanyPots to load an invalid block for the crop. This has now been fixed in 13.0.8.
